examples/plot_global_reject.py

```diff
@@ -50,10 +50,10 @@
 # (RMSE) values at the candidate thresholds, we will use the function
 # :func:`autoreject.validation_curve`.
 
-from autoreject import GlobalAutoReject, validation_curve  # noqa
+from autoreject import _GlobalAutoReject, validation_curve  # noqa
```

Comment:
@agramfort @dengemann we cannot expose private classes / methods in examples. What do you suggest for this?

Reply:
I suppose we can remove this example. It is reproducing Figure 1 in our paper ...